### ISSUE TITLE
[insights] add filter for has alt text (or not)

### ISSUE THREAD

**Issue by btopro** [elmsln/issues#1054]
Add a filter for alt text and if it exists when doing insights on media
- [x] bring along `alt` as we run the service
- [x] render it on images
- [x] allow for detecting / flagging "blank alt text" with a warning symbol
- [x] word "image" in the alt data, warning symbol
- [x] no alt text, error
- [x] alt text same as file name, error
- [x] backend detection and front end detection not lining up count wise...
